plugin/reload/README.md

```diff
@@ -99,6 +99,7 @@ CoreDNS v1.7.0 and later does parse the Corefile and supports detecting changes
  If monitoring is enabled (via the *prometheus* plugin) then the following metric is exported:
 
 * `coredns_reload_failed_count_total{}` - counts the number of failed reload attempts.
+* `coredns_reload_version_info{}` - record the hash value during reload.
```

Comment:
please add the labels as well, e.g. from *forward*
`* `coredns_forward_response_rcode_count_total{to, rcode}` - count of RCODEs per upstream.`